src/workshops/HDA_Flowsheet_Optimization/flowsheet_kinetic_OUU.py

```diff
@@ -0,0 +1,404 @@
+        m.fs.translator.inlet.pressure[0])
+
+    m.fs.translator.eq_mole_frac_benzene = Constraint(
+        expr=m.fs.translator.outlet.mole_frac_comp[0, "benzene"] ==
+        m.fs.translator.inlet.flow_mol_phase_comp[0, "Liq", "benzene"] /
+        (m.fs.translator.inlet.flow_mol_phase_comp[0, "Liq", "benzene"] +
+         m.fs.translator.inlet.flow_mol_phase_comp[0, "Liq", "toluene"]))
+
+    m.fs.translator.eq_mole_frac_toluene = Constraint(
+        expr=m.fs.translator.outlet.mole_frac_comp[0, "toluene"] ==
+        m.fs.translator.inlet.flow_mol_phase_comp[0, "Liq", "toluene"] /
+        (m.fs.translator.inlet.flow_mol_phase_comp[0, "Liq", "benzene"] +
+         m.fs.translator.inlet.flow_mol_phase_comp[0, "Liq", "toluene"]))
+
+    m.fs.reaction_params = reaction_props.HDAReactionParameterBlock(
+            default={"property_package": m.fs.thermo_params})
+
+    m.fs.M101 = Mixer(default={
+        "property_package": m.fs.thermo_params,
+        "inlet_list": ["toluene_feed", "hydrogen_feed", "vapor_recycle"]})
+
+    m.fs.H101 = Heater(default={"property_package": m.fs.thermo_params,
+                                "has_pressure_change": False,
+                                "has_phase_equilibrium": True})
+
+    m.fs.R101 = CSTR(
+                default={"property_package": m.fs.thermo_params,
+                         "reaction_package": m.fs.reaction_params,
+                         "has_heat_of_reaction": True,
+                         "has_heat_transfer": True,
+                         "has_pressure_change": False})
+
+    m.fs.F101 = Flash(default={"property_package": m.fs.thermo_params,
+                               "has_heat_transfer": True,
+                               "has_pressure_change": True})
+
+    m.fs.S101 = Splitter(default={"property_package": m.fs.thermo_params,
+                                  "ideal_separation": False,
+                                  "outlet_list": ["purge", "recycle"]})
+
+    m.fs.C101 = PressureChanger(default={
+                "property_package": m.fs.thermo_params,
+                "compressor": True,
+                "thermodynamic_assumption":
+                ThermodynamicAssumption.isothermal})
+
+    m.fs.s03 = Arc(source=m.fs.M101.outlet, destination=m.fs.H101.inlet)
+    m.fs.s04 = Arc(source=m.fs.H101.outlet, destination=m.fs.R101.inlet)
+    m.fs.s05 = Arc(source=m.fs.R101.outlet, destination=m.fs.F101.inlet)
+    m.fs.s06 = Arc(source=m.fs.F101.vap_outlet, destination=m.fs.S101.inlet)
+    m.fs.s08 = Arc(source=m.fs.S101.recycle, destination=m.fs.C101.inlet)
+    m.fs.s09 = Arc(source=m.fs.C101.outlet,
+                   destination=m.fs.M101.vapor_recycle)
+
+    m.fs.s10 = Arc(source=m.fs.F101.liq_outlet,
+                   destination=m.fs.translator.inlet)
+    m.fs.s11 = Arc(source=m.fs.translator.outlet,
+                   destination=m.fs.pre_heater.inlet)
+
+    TransformationFactory("network.expand_arcs").apply_to(m)
```

Comment:
We can't do it yet because of the bug/issue with the SD tool, but keep in mind that we will have to move the column declaration up here for the actual paper.